bigquery/nox.py

```diff
@@ -65,7 +65,7 @@ def default(session):
 
 
 @nox.session
-@nox.parametrize('py', ['2.7', '3.5', '3.6'])
+@nox.parametrize('py', ['2.7', '3.5', '3.6', '3.7'])
```

Comment:
Add the same to `system`?

Reply:
I guess we don't have 3.5 on system, either, nevermind.

Reply:
Right, we need to keep `system` as snappy as possible.  The chance that 3.x will break a system test which 3.y passes is pretty minimal.

Reply:
System should likely be set to 2.7, 3.7 though.

Reply:
3.7 is still beta.